youtube_dl/extractor/common.py

```diff
@@ -826,6 +826,24 @@ def _get_netrc_login_info(self, netrc_machine=None):
             except (IOError, netrc.NetrcParseError) as err:
                 self._downloader.report_warning(
                     'parsing .netrc: %s' % error_to_compat_str(err))
+        
+        if self._downloader.params.get('usekeyring', False):
+            try:
+                import gi
+                gi.require_version('Secret', '1')
+                from gi.repository import Secret
+                LIBSECRET_SCHEMA = Secret.Schema.new("io.github.rg3.youtube-dl.Store",Secret.SchemaFlags.DONT_MATCH_NAME,{"extractor": Secret.SchemaAttributeType.STRING,"app": Secret.SchemaAttributeType.STRING})
```

Comment:
Better to wrap such long lines.

Reply:
Wrapped in https://github.com/rg3/youtube-dl/pull/14674/commits/3be4a1db6b6e144390cb337888daba6c4b52c852